extensions/css-language-features/server/src/cssServerMain.ts

```diff
@@ -50,6 +52,8 @@ let scopedSettingsSupport = false;
 let foldingRangeLimit = Number.MAX_VALUE;
 let workspaceFolders: WorkspaceFolder[];
 
+const languageServices: { [id: string]: LanguageService } = {};
```

Comment:
I guess that's not really a necessary change.

Reply:
Doing this so I can locally scope the data variables. I don't want the `customData` to be on top level.

Reply:
So you say vs code can complete css variables by setting this setting